tests/lint/test/data/library-missing-url-path.fmf

```diff
@@ -0,0 +1,5 @@
+summary: Library requirement missing both url and path
+test: /bin/true
+require:
+  - type: library
+    name: /some/library
```

Comment:
Please this test case in https://github.com/teemtee/tmt/blob/main/tests/lint/test/test.sh as well